diff --git a/node_modules/@cucumber/cucumber/lib/formatter/helpers/usage_helpers/index.js b/node_modules/@cucumber/cucumber/lib/formatter/helpers/usage_helpers/index.js
index 35339d7..c6001c0 100644
--- a/node_modules/@cucumber/cucumber/lib/formatter/helpers/usage_helpers/index.js
+++ b/node_modules/@cucumber/cucumber/lib/formatter/helpers/usage_helpers/index.js
@@ -86,10 +86,7 @@ function buildResult(mapping) {
         .map((stepDefinitionId) => {
         const { matches, ...rest } = mapping[stepDefinitionId];
         const sortedMatches = matches.sort((a, b) => {
-            if (a.duration === b.duration) {
-                return a.text < b.text ? -1 : 1;
-            }
-            return normalizeDuration(b.duration) - normalizeDuration(a.duration);
+            return a.text.localeCompare(b.text);
         });
         const result = { matches: sortedMatches, ...rest };
         const durations = matches
@@ -101,7 +98,7 @@ function buildResult(mapping) {
         }
         return result;
     })
-        .sort((a, b) => normalizeDuration(b.meanDuration) - normalizeDuration(a.meanDuration));
+        .sort((a, b) => a.uri.localeCompare(b.uri));
 }
 function getUsage({ stepDefinitions, eventDataCollector, }) {
     const mapping = buildMapping({ stepDefinitions, eventDataCollector });
